oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1410,7 +1412,10 @@ void pldm::responder::oem_ibm_platform::Handler::handleBootTypesAtPowerOn()
             setBootTypesBiosAttr(restartCause);
         }
         catch (const std::exception& e)
-        {}
+        {
+            error("Failed to call the D-Bus Method ERROR={ERR}", "ERR",
```

Comment:
Pls change "Failed to call the D-Bus Method" to "Failed to set the D-bus property for the Host restart reason"